src/Scoringline/SendinblueApi/Model/Email.php

```diff
@@ -0,0 +1,311 @@
+    private $bcc = [];
+
+    /**
+     * @var array
+     */
+    private $attachment = [];
+
+    /**
+     * @var array
+     */
+    private $inlineImage = [];
+
+    /**
+     * @param array $to
+     * @return Email
+     */
+    public function setTo(array $to)
+    {
+        $this->to = $to;
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getTo()
+    {
+        return $this->to;
+    }
+
+    /**
+     * @param array $from
+     * @return Email
+     */
+    public function setFrom(array $from)
+    {
+        $this->from = $from;
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getFrom()
+    {
+        return $this->from;
+    }
+
+    /**
+     * @param array $subject
+     * @return Email
+     */
+    public function setSubject($subject)
+    {
+        $this->subject = $subject;
+
+        return $this;
+    }
+
+    /**
+     * @return string
+     *
+     */
+    public function getSubject()
+    {
+        return $this->subject;
+    }
+
+    /**
+     * @param array $text
+     * @return Email
+     */
+    public function setText($text)
+    {
+        $this->subject = $text;
+
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getText()
+    {
+        return $this->text;
+    }
+
+    /**
+     * @param array $html
+     * @return Email
+     */
+    public function setHtml($html)
+    {
+        $this->subject = $html;
+
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getHtml()
+    {
+        return $this->html;
+    }
+
+    /**
+     * @param array $headers
+     * @return Email
+     */
+    public function setHeaders(array $headers)
+    {
+        $this->headers = $headers;
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getHeaders()
+    {
+        return $this->headers;
+    }
+
+    /**
+     * @param array $replyTo
+     * @return Email
+     */
+    public function setReplyTo(array $replyTo)
+    {
+        $this->replyTo = $replyTo;
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getReplyTo()
+    {
+        return $this->replyTo;
+    }
+
+    /**
+     * @param array $cc
+     * @return Email
+     */
+    public function setCc(array $cc)
+    {
+        $this->cc = $cc;
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getCc()
+    {
+        return $this->cc;
+    }
+
+    /**
+     * @param array $bcc
+     * @return Email
+     */
+    public function setBcc(array $bcc)
+    {
+        $this->bcc = $bcc;
+
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getBcc()
+    {
+        return $this->bcc;
+    }
+
+    /**
+     * @param array|object $attachmentData
+     * @return Email
+     */
+    public function setAttachment($attachmentData)
```

Comment:
The problem is that your attribute name is wrong: you can have many attachments, so you should add a `s` to the variable to be more clear.

And then, on `setAttachments` you should take an array that contains `File` objects, or strings.

The addAttachment method will allow the user to add attachments one by one.